lstchain/datachecks/containers.py

```diff
@@ -282,6 +308,40 @@ def fill_pixel_wise_info(self, table, mask, histogram_binnings,
                      bins=histogram_binnings.hist_pixelchargespectrum)
         self.hist_pixelchargespectrum = counts
 
+        # Find bright stars (mag<=8 within 3 deg of telescope pointing):
+
+        # Just use the time in the middle of the subrun, from the sampled times:
+        sampled_times = self.dragon_time
+        obstime = Time(sampled_times[int(len(sampled_times)/2)],
+                       scale='utc', format='unix')
+        horizon_frame = AltAz(location=location, obstime=obstime)
+        pointing = SkyCoord(az=self.mean_az_tel*u.rad,
+                            alt=self.mean_alt_tel*u.rad,
+                            frame=horizon_frame)
+        bright_stars = get_bright_stars(pointing=pointing, radius=3*u.deg,
+                                        magnitude_cut=8)
+        camera_frame = CameraFrame(telescope_pointing=pointing,
```

Comment:
I would do this in `TelescopeFrame`, not `CameraFrame` since then you can directly use:

```
telescope_frame = TelescopeFrame(telescope_pointing=pointing, obstime=obstime, location=location)
bright_stars['tel_frame'] = bright_stars['ra_dec'].transform_to(telescope_frame)
pixels_tel_frame = SkyCoord(x=geom.pix_x, y=geom.pix_y, frame=geom.frame).transform_to(telescope_frame)

# apply this relative shift here?
# Isn't this just a smaller focal length than the nominal one?

separation = pixels_tel_frame.separation(bright_stars['tel_frame'][:, np.newaxis])
n_stars = np.count_nonzero(separation < r_around_star, axis=1)
```

Reply:
This would however not apply the "scaling" due to the coma aberration, right? I want to have both shifted star positions, and larger effective radius as we move out in the fov.  Will give it a try.

Reply:
You can just apply that to the pixel coordinates when creating the camera frame  `SkyCoord(x=cam.pix_x * scaling`, y=cam.pix_y * scaling, ...)`

Reply:
Edit: or better like discussed in slack, just scale the focal length in the `CameraFrame(focal_length = 28 * u.m  * scaling, ...)`

Reply:
Edit: the earlier comment about scaling the pixel positions was wrong

Reply:
> Edit: the earlier comment about scaling the pixel positions was wrong

I think scaling the pixel coordinates by  1 / 1.0466 and keeping the original focal length for the transformations to angles is fully equivalent.